docs/getting_started/QuickstartTutorial.md

```diff
@@ -14,13 +14,13 @@ To do so, you will
 * Create new commits and issues
 * Watch as the metrics and trends change as new versions roll out**\***
 
-> **Note:** More information about the four key DORA metrics can be found in our [Outcomes documentation](./philosophy/outcomes/index.md).
+> **Note:** More information about the four key DORA metrics can be found in our [Outcomes documentation](#philosophy/outcomes/Overview/).
```

Comment:
this link did not work

Reply:
Fixed, great catch. Actually mkdocs serve does not give warning as it's just link to local tag.